test/integration/roles/test_ecs_ecr/tasks/main.yml

```diff
@@ -5,6 +5,10 @@
 - block:
     - name: When creating with check mode
       ecs_ecr: name='{{ ecr_name }}' region='{{ ec2_region }}'
+      environment:
```

Comment:
@willthames Thanks for looking at this. Since we're repeating the credentials for ever single task, I wish there was an easy way to extrapolate them. I definitely don't want to set environment variables if we aren't specifically testing them. I'm unsure how to approach this problem or how I feel about using profile. Our integration test [credentials.yml](https://github.com/ansible/ansible/blob/devel/test/integration/credentials.template) doesn't currently have an ec2_profile var so we would need to add that, update [documentation](https://github.com/ansible/ansible/blob/devel/test/integration/README.md#cloud-tests) and update all the tests. I guess we could also use the shell module to create a testing profile with the credentials.yml file and remove it afterward?